redbot/cogs/mod/settings.py

```diff
@@ -199,3 +202,19 @@ class ModSettings(MixinMeta):
                     command=f"{ctx.prefix}unban"
                 )
             )
+
+    @modset.command()
+    @commands.guild_only()
+    async def defaultdays(self, ctx: commands.Context, days: int = 0):
+        """Set the default days worth of messages to be deleted when a user is banned.
+        The amount of days has to be between 0 and 7.
```

Comment:
This needs an empty line here to show up properly in help and it should be "number of days" not "amount of days" and I think "must be" works better here than "has to"
```suggestion

        The number of days must be between 0 and 7.
```